auditbeat/module/file_integrity/config.go

```diff
@@ -160,6 +177,11 @@ nextHash:
 	if err != nil {
 		errs = append(errs, fmt.Errorf("invalid scan_rate_per_sec value: %w", err))
 	}
+
+	if c.Backend != "" && runtime.GOOS != "linux" {
```

Comment:
As a user I would expect to be able to deploy `backend: auto` in all of my configurations despite the OS.